samples/ChatSample/Hubs/Chat.cs

```diff
@@ -18,7 +18,7 @@ public override Task OnConnectedAsync()
         {
             if (!Context.User.Identity.IsAuthenticated)
             {
-                Context.Connection.Channel.Dispose();
+                //Context.Connection.Channel.Dispose();
```

Comment:
`Context.Connection` is now the `HubConnection` so you don't have direct access to the `Channel`